MAINTAINERS

```diff
@@ -124,8 +124,90 @@ the relevant operators.
 
 * If the change affects the governance, philosophy, goals or principles of the project,
 it must be approved by BDFL.
+
+* A pull request can be in 1 of 5 distinct states, for each of which there is a corresponding label
+that needs to be applied. `Rules.review.states` contains the list of states with possible targets
+for each.
 """
 
+		# Triage
+		[Rules.review.states.nolabel]
```

Comment:
rename to `0-triage` and explain that these are incoming PRs automatically labeled with `0-triage`. The reason for this is to circumvent a limitation with a tool we'd like to experiment with: waffle.io. If it does not suit our needs, we can revisit this particular label vs no-label for incoming PRs.